server/controllers/user.js

```diff
@@ -2,7 +2,7 @@ import isEmpty from "lodash.isempty";
 import { comparePassword } from "../helpers/password";
 import { serverError, httpResponse } from "../helpers/http";
 import { generateJWT, decodeJWT, getJWTConfigs } from "../helpers/jwt";
-import { userResponse } from "../helpers/userResponse";
+import { userResponse, userProfileResponse } from "../helpers/userResponse";
```

Comment:
I don't like this has two functions userResponse and userProfileResponse in a helper named after one of the functions. Take these 2 functions into the user helpers